README.md

```diff
@@ -52,7 +52,7 @@ $$
 \end{split}
 $$
 
-Vector inequalities apply coordinate by coordinate. The function returns the solution $x^\*$ found by the solver, or ``None`` in case of failure/unfeasible problem. For most solvers, the matrix $P$ should be [positive definite](https://en.wikipedia.org/wiki/Definite_symmetric_matrix).
+Vector inequalities apply coordinate by coordinate. The function returns the solution $x^\*$ found by the solver, or ``None`` in case of failure/unfeasible problem. For all the solvers, the matrix $P$ should be [positive  semi-definite](https://en.wikipedia.org/wiki/Definite_symmetric_matrix) which implies a convex problem. For some solvers the more strict of condition of positive definiteness is required.
```

Comment:
I agree it's an improvement to mention both :+1: 

Suggesting one more iteration:
```suggestion
Vector inequalities apply coordinate by coordinate. The function returns the solution $x^\*$ found by the solver, or ``None`` in case of failure/unfeasible problem. All solvers require the problem to be convex, meaning the matrix $P$ should be [positive semi-definite](https://en.wikipedia.org/wiki/Definite_symmetric_matrix). Some solvers further require the problem to be strictly convex, meaning $P$ should be positive definite.
```